### DISCUSSION TITLE
Backslashes in VSCode with Bash terminal

### DISCUSSION THREAD

**Discussion by jerone** [JanDeDobbeleer/oh-my-posh#5464] [Troubleshoot]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

When using the Bash terminal within VSCode, all commands (except for the first one) are rendered with extra backslashes in the view.

![image](https://github.com/user-attachments/assets/048056e2-ae0e-4491-b41a-deaf5909fac9)

It only happens in VSCode. Not in Windows Terminal with Bash.
It only happens with Bash in VSCode. Not with Powershell for example.
It also happens on VSCode Insiders.

VSCode config:
```json
{
  "terminal.integrated.fontFamily": "MesloLGM Nerd Font",
}
```

OhMyPosh: 23.6.3
VSCode: 1.92.2
VSCode Insiders: 1.93.0
Windows: 10 Pro

### Theme

Default.

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

bash

### Log output

```Shell
Print the prompt in debug mode.

Usage:
  oh-my-posh prompt debug [bash|zsh|fish|powershell|pwsh|cmd|nu|tcsh|elvish|xonsh] [flags]

Flags:
  -h, --help           help for debug
  -p, --plain          plain text output (no ANSI)
      --pwd string     current working directory
      --shell string   the shell to print for

Global Flags:
  -c, --config string   config file path
```

**Comment by JanDeDobbeleer** [comment]
@jerone pretty certain this is due to their shell integrations feature. Oh-my-posh has no contextual vscode logic, so something external should be influencing this. Try disabling that inside vscode, reload and check again.
